### ISSUE TITLE
Change default mesh object

### ISSUE THREAD

**Issue by kakachiex2** [gokayfem/ComfyUI-Texture-Simple#1]
@gokayfem it is possible to change the default model and add my own

**Comment by gokayfem** [comment]
yes im planning to add this kind of functionality also i want to make a GLTF viewer as a seperate node. not sure when but soon.
